src/test/scala/com/cognite/sdk/scala/v1/DataPointsTest.scala

```diff
@@ -94,6 +95,15 @@ class DataPointsTest extends SdkTest with DataPointsResourceBehaviors {
     }
   }
 
+  val sumsOnly = client.dataPoints.queryAggregatesById(
```

Comment:
Not sure if this is good enough to catch the error, really. We should perhaps have a test that fetches all aggregates from a fixed time series for a fixed time window, and compares all of them to known values (retrieved from the Python SDK or using Postman).